AlertaDengue/ad_main/celeryapp.py

```diff
@@ -1,13 +1,12 @@
 import os
 
 from celery import Celery
-from django.conf import settings
 
 # This makes sure celery knows where settings are, so we don't need to set it
 # when calling it from the command line
 os.environ.setdefault("DJANGO_SETTINGS_MODULE", "ad_main.settings")
 
 
 app = Celery("ad_main", include=["dbf.tasks"])
-app.config_from_object(settings, namespace="CELERY")
+app.config_from_object("ad_main.celery_settings", namespace="CELERY")
```

Comment:
I think that before merging these changes, we need to make some DBF upload tests in staging.